templates/nginx/vhost.conf

```
@@ -13,7 +13,7 @@ server {
     ssl_certificate_key {{ bbb_ssl_key }};
     ssl_session_cache shared:SSL:10m;
     ssl_session_timeout 1d;
-    ssl_protocols TLSv1.2;
+    ssl_protocols TLSv1.2 TLSv1.3;
```

When the use of the official nginx-repo is selectable, one should make 'TLSv1.3' depend on that choice.

---

TLSv1.3 support is included in Nginx since version 1.13. I don't understand why TLS 1.3 was not enabled by default in the BBB templates. So I think that this should be the default anyways. 

> "Nginx version 1.13.0 or greater built against OpenSSL 1.1.1 or greater."

---

Oh yes, then we could even prefer TLS1.3 over TLS1.2:
```suggestion
    ssl_protocols TLSv1.3 TLSv1.2;
```

---

> TLSv1.3 support is included in Nginx since version 1.13. I don't understand why TLS 1.3 was not enabled by default in the BBB templates. So I think that this should be the default anyways.
> 
> > "Nginx version 1.13.0 or greater built against OpenSSL 1.1.1 or greater."

I think historically. In ubuntu 16, nginx was not compiled against openssl 1.1.1.

---

Maybe, but now, we are on Ubuntu 18.04.
I tested it. Even with ubuntu-packaged nginx (1.14) you can use TLSv1.3, since openssl 1.1.1 is installed on Ubuntu 18.04